src/blocks/weather/met_no.rs

```diff
@@ -194,9 +201,11 @@ fn weather_to_icon(weather: &str) -> WeatherIcon {
         | "lightsleetandthunder"
         | "lightrainandthunder"
         | "lightsnowandthunder"
-        | "lightssleetshowersandthunder"
-        | "lightssnowshowersandthunder"
-        | "lightrainshowersandthunder" => WeatherIcon::Thunder,
+        | "lightssleetshowersandthunder" // There's a typo in the api it will be fixed in the next version to the following entry
+        | "lightsleetshowersandthunder"
```

Comment:
Can you please point me to where you found this info?

Reply:
https://api.met.no/weatherapi/locationforecast/2.0/documentation

> Note: There is a typing error in lightssleetshowersandthunder and lightssnowshowersandthunder (extra "s" after "light"). Unfortunately, correcting this would mean breaking existing applications, so it has been postponed to the next version of weathericon/locationforecast.